tasks/13.txt

```diff
@@ -1,3 +1,18 @@
 Create a SQL query to count current and discontinued products:
 
-| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
\ No newline at end of file
+| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
+
+//task13
+db.products.aggregate([
+{ $group:
+      {
+        _id: 0,
+        TotalOfCurrentProducts: { 
+           $sum: { $cond: {if:{ $eq: [ "$Discontinued", 0 ] }, then:1 , else:0 } }
+           },
+        TotalOfDiscontinuedProducts : { 
+           $sum: { $cond: {if:{ $eq: [ "$Discontinued", 1 ] }, then:1 , else:0 } }
+           }
+      }
+}
+])
```

Comment:
Result is correct, but query will be slow. We have a better way to get the result. 
Also please note we not always have to follow requirements strongly. I mean result format. We can do some mapping on server side in real cases if needed.